cleanlab/models/keras.py

```diff
@@ -36,7 +36,7 @@
 """
 
 import tensorflow as tf
-import keras
+import keras  # type: ignore
```

Comment:
Ignoring type here because keras has no stubs available and will raise error (ref [here](https://github.com/huiwengoh/cleanlab/actions/runs/4084016149/jobs/7040218721))